.babelrc

```diff
@@ -1,7 +1,4 @@
 {
-  "presets": [["env", {
-    "targets": {
-      "browsers": ["last 2 versions", "ie 9-11"]
-    }
-  }]]
+  presets: ['es3', ['es2015', { loose: true, modules: false }]],
+  plugins: ['external-helpers', 'transform-object-assign']
 }
```

Comment:
why not preset env?

Reply:
we probably could but since we're targeting IE11 and up, es2015 preset is kind of easier.

Reply:
either one is fine. The babel team would probably prefer us to use the babel-preset-env, though.

Reply:
easier for what?

especially because we are targeting IE11 and up if we use env and target browsers properly we can end up with less compiled features and build size will be smaller and work better (on supported browsers)

also env preset can decide if we need es3 stuff automatically by target browsers